src/js/utils/FormUtil.js

```diff
@@ -0,0 +1,89 @@
+function cloneDefinition(array) {
+  return array.map(function (object) {
+    if (Array.isArray(object)) {
+      return object.slice();
+    }
+
+    if (typeof object === 'object') {
+      return Object.assign({}, object);
+    }
+
+    return object;
+  });
+}
```

Comment:
this no longer exists. i ported over `deepCopy` from marathon-ui (good stuff guys)